fib ms5/js/fibo.js

```
@@ -0,0 +1,37 @@
+
+// milestone 4//
+const FB_URL = "http://localhost:5050/fibonacci/";
+const button = document.getElementById("submit");
+const output = document.getElementById("outputN");
+const input = document.getElementById("inputN");
+const spinnerOn = document.getElementById("spinner");
+ //---in html-+ out to server 
+
+button.addEventListener("click", (outToServer));
+
+function outToServer() {
+  spinnerOn.classList.remove('d-none')
+  const sendInfo = input.value;
+
+  if (sendInfo < 51) {
+    fetch(`${FB_URL}${sendInfo}`)
+    
+      .then(response => {
+        if (!response.ok) {
+          response.text().then((data) => {
```

You should throw a new error with the data here and display it in the .catch: 
`throw new Error(data)`

`.catch(err => {
 output.innerHTML = `Server Error: ${data}`;
})`